src/fragments/lib/datastore/android/advanced-workflows/custom-primary-key/delete-snippet.mdx

```diff
@@ -0,0 +1,143 @@
+In addition to deleting by the model type (or `Predicates.ALL`), a record with a custom primary key can be deleted the following ways:
+
+
+With a predicate:
+
+Custom Primary Key
+
+<Block name="Custom Primary Key">
+<BlockSwitcher>
+<Block name="Java">
+
+```java
```

Comment:
+1

Reply:
@poojamat - Is there anything we can do to potentially format the code examples so that horizontal scroll is not necessary? Perhaps less initial indentation?